packages/website/src/components/Footer/index.tsx

```diff
@@ -126,6 +126,8 @@ const moreBlock: IBlockItem = {
   ]
 }
 
+const getThisYear = (): String => new Date().getFullYear().toString()
```

Comment:
```suggestion
const getThisYear = (): string => new Date().getFullYear().toString()
```
`String` 和 `string` 还是有点区别的
https://www.typescriptlang.org/docs/handbook/basic-types.html#about-number-string-boolean-symbol-and-object

Reply:
好的，感谢大佬提醒（bgm38）

Reply:
诶，没看见改动啊，忘 push 了？

Reply:
> 诶，没看见改动啊，忘 push 了？

没注意，给忘了emm.

Reply:
> 诶，没看见改动啊，忘 push 了？

加上了(bgm42)